### ISSUE TITLE
Safari issue in RWD beta - learn more about CSS Pseudo Selectors

### ISSUE THREAD

**Issue by marks1974** [freeCodeCamp/freeCodeCamp#44801]
## Affected page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-2

## Your code

```

  <body>
    <div id="sheet"></div>
    <footer>Last Updated: December 2021</footer>
  </body>


```

## Expected behavior

The above code does not pass when submitted in Safari. I checked the same step using Firefox and it passed ok. 
## Screenshots
![screenshot_150](https://user-images.githubusercontent.com/13849426/149678965-75c1fbb3-479a-4220-9073-9b8199bfd0cb.png)

## System

macOS 12.1
Safari 15.2

**Comment by moT01** [comment]
I couldn't reproduce this on Safari 13.1.2. The test that is failing for you, @marks1974, is checking for a single `div` element: `assert(document.querySelectorAll('div')?.length === 1);`. Seems like a strange one to fail. There must be quite a few tests like that throughout that would be causing you problems. A quick search shows as many as 127:

![Screen Shot 2022-01-20 at 1 32 37 PM](https://user-images.githubusercontent.com/20648924/150409424-4d7f2e5e-ec45-4aa5-bbc7-f79e26c0055d.png)

Maybe you just haven't come across any of the others I guess. Another one is [step 12 of the magazine project](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-grid-by-building-a-magazine/step-12)

Want to give both of these (my link and yours) another try and make sure it wasn't a fluke @marks1974?

**Comment by marks1974** [comment]
I just tried step 2 again and this time it passed. I honestly thought it was a bit odd that it did not initially pass as shown by my screen shots.

The funny thing is that earlier in the week I was doing a different part of the beta and initially one of the steps (I can't remember which) would not pass for me in Safari but was fine in Firefox. I started to write a new issue on Github and was about to hit submit but before doing so I decided to test it again. After trying about half a dozen times, it finally worked. As it was being a bit hit and miss in Safari I switched to using Firefox which has been mostly bug free. Having said that I had a problem earlier today in Firefox when I was trying to submit some of the steps in the penguin project which were asking for linear-gradients (I think it was steps 8 and 14). They would not pass in Firefox but oddly enough they passed in Safari.
